src/sage/combinat/designs/difference_family.py

```diff
@@ -2193,6 +2193,316 @@ def generate_subset(indices, H):
     return G, [S1, S2, S3, S4]
 
 
+def _construct_gs_difference_family_from_full(S1, S2, mu):
+    r"""
+    Construct a spin type Goethals-Seidel difference family given the first two
+    sets.
+
+    This construction is described in [Djo2024]_. Given the first two sets
+    `S_1, S_2`, and the multiplier `\mu`, the last two sets are computed as
+    `S_3 = \mu S_2`, `S_4 =\mu S_3`.
+
+    The sets should contain elements of a group modulo `n`.
```

Comment:
what does this mean? invertible elements in $\mathbb{Z}/n\mathbb{Z}$ ? Something else?

Reply:
I've changed it to make it clearer.